std/bytes/mod.ts

```diff
@@ -45,36 +53,52 @@ export function findLastIndex(a: Uint8Array, pat: Uint8Array): number {
   return -1;
 }
 
-/** Check whether binary arrays are equal to each other **/
-export function equal(a: Uint8Array, match: Uint8Array): boolean {
-  if (a.length !== match.length) return false;
-  for (let i = 0; i < match.length; i++) {
-    if (a[i] !== match[i]) return false;
-  }
-  return true;
+/**
+ * Check whether binary arrays are equal to each other.
+ * @param source first array to check equality
+ * @param match second array to check equality
+ */
+export function equal(source: Uint8Array, match: Uint8Array): boolean {
+  return String(source) === String(match);
```

Comment:
Comparing by coercing to strings is slower in every imaginable way.  See: https://jsperf.com/uint8compare/1

Why are you attempting to change this?

Reply:
I think it was cleaner, didn't check for performance actually.
reverted
thanks for notice that 👍